topics/statistics/tutorials/iwtomics/tutorial.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: tutorial_hands_on
+topic_name: statistics
+tutorial_name: iwtomics
+---
+
+# Introduction
+{:.no_toc}
+
+IWTomics implements the Interval-Wise Testing (IWT) for omics data. This
+inferential procedure tests for differences in "Omics" data between two groups
+of genomic regions, and does not require fixing location and scale at the
+outset.
+
+In this tutorial we use real data about endogenous retroviruses (ERVs) in mouse.
+In particular, we compare recombination hotspots in the flanking regions of fixed ETn versus control regions.
+The dataset contains two region datasets "ETn fixed", "Control" and one feature "Recombination hotspots content". The region dataset "ETn fixed" contains 1296 genomic regions of 64 kb surrounding fixed ETns elements (32-kb flanking sequences upstream and 32-kb flanking sequences downstream of each element). The region dataset "Control" contains 1142 regions of 64 kb without elements, used as control in the test. The regions are aligned around their center (i.e. around the ETn integration sites).
+
+Recombination hotspots measurements are associated to each "ETn fixed" and "Control" region. In particular, this feature is measured in 1-kb windows, so that each region is associated to a recombination hotspots curve made of 64 values. The measurement used is the feature content, i.e. the fraction of the 1-kb window that is covered by recombination hotspots
+
+The data we use in this tutorial is available at [Zenodo](https://doi.org/10.5281/zenodo.1184682).
+
+> ### Agenda
+>
+> IWTomics is composed of three steps (corresponding to three tools):
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Step 1: Loading and pre-processing
+
+> The first tool (IWTomics Load Smooth and Plot) imports a collection of genomic region datasets, and associates to each region multiple genomic feature measurements. It allows to align the regions in multiple ways (center, left, right or scale alignment), to smooth the feature curves (possibly filling gaps in the measurements) and to create a graphical representation of the feature measurements in each region datasets (aligned curves or pointwise quantile curves).
+
+> ### {% icon hands_on %} Hands-on: Get the data
+> Load the example dataset which will be used for the tutorial.
+> 1. Upload Fixed ETn (`ETn_fixed.bed`) and Control (`Control.bed`) regions, as well as Recombination Hotspot (`Recombination_hotspots.txt`) content.
+> 2. Upload header files for regions (regions_header.tabular) and features (features_header.tabular).
+>    > ### {% icon tip %} Tip: Make sure to select the rigth data type
+>    > * Select `tabular` type.
+>    {: .tip}
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Pre-process data and create pointwise boxplot
+> 1. Execute the IWTomics Tool **Load, Smooth and Plot**.
```

Comment:
Could you format with something like: `**Load, Smooth and Plot** {% icon tool %}: Run **Load, Smooth and Plot** with` ?
We try to keep some homogenization (http://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#improving-the-learning-experience)

Reply:
done